AI/scripts/download_weights.py

```
@@ -0,0 +1,190 @@
+        return []
+    data = resp.json()
+    files = data.get("files", [])
+    return files
+
+
+def download_file(slug: str, file_name: str, dst_path: str) -> bool:
+    """Kaggle API로 특정 파일 다운로드 (스트리밍)"""
+    owner, kernel = slug.split("/")
+    url = f"https://www.kaggle.com/api/v1/kernels/output/{owner}/{kernel}?fileName={file_name}"
+    
+    with requests.get(url, auth=(KAGGLE_USERNAME, KAGGLE_KEY), stream=True) as resp:
```

Ruff (0.15.7)

[error] Probable use of `requests` call without timeout

(S113)